Package.swift

```diff
@@ -0,0 +1,127 @@
+            cxxSettings: [
+                .headerSearchPath("../SentryCrash/Installations"),
+                .headerSearchPath("../SentryCrash/Recording"),
+                .headerSearchPath("../SentryCrash/Recording/Monitors"),
+                .headerSearchPath("../SentryCrash/Recording/Tools"),
+                .headerSearchPath("../SentryCrash/Reporting/Filters"),
+            ],
+            linkerSettings: [
+                .linkedLibrary("z"),
+                .linkedLibrary("c++"),
+            ]
+        ),
+
+        .target(
+            name: "SentryCrash/Installations",
+            path: "Sources/SentryCrash/Installations",
+            publicHeadersPath: ".",
+            cxxSettings: [
+                .headerSearchPath("../Recording"),
+                .headerSearchPath("../Recording/Monitors"),
+                .headerSearchPath("../Recording/Tools"),
+                .headerSearchPath("../Reporting/Filters"),
+                .headerSearchPath("../Reporting/Tools"),
+            ]
+        ),
+
+        .target(
+            name: "SentryCrash/Recording",
+            path: "Sources/SentryCrash/Recording",
+            exclude: [
+                "Monitors",
+                "Tools",
+            ],
+            publicHeadersPath: ".",
+            cxxSettings: [
+                .headerSearchPath("Tools"),
+                .headerSearchPath("Monitors"),
+                .headerSearchPath("../Reporting/Filters"),
+            ]
+        ),
+
+        .target(
+            name: "SentryCrash/Recording/Monitors",
+            path: "Sources/SentryCrash/Recording/Monitors",
+            publicHeadersPath: ".",
+            cxxSettings: [
+                .define("GCC_ENABLE_CPP_EXCEPTIONS", to: "YES"),
+                .headerSearchPath(".."),
+                .headerSearchPath("../Tools"),
+                .headerSearchPath("../../Reporting/Filters"),
+            ]
+        ),
+
+        .target(
+            name: "SentryCrash/Recording/Tools",
+            path: "Sources/SentryCrash/Recording/Tools",
+            publicHeadersPath: ".",
+            cxxSettings: [
+                .headerSearchPath(".."),
+            ]
+        ),
+
+        .target(
+            name: "SentryCrash/Reporting/Filters",
+            path: "Sources/SentryCrash/Reporting/Filters",
+            exclude: [
+                "Tools",
+            ],
+            publicHeadersPath: ".",
+            cxxSettings: [
+                .headerSearchPath("Tools"),
+                .headerSearchPath("../../Recording/Tools"),
+            ]
+        ),
+
+        .target(
+            name: "SentryCrash/Reporting/Filters/Tools",
+            path: "Sources/SentryCrash/Reporting/Filters/Tools",
+            publicHeadersPath: "."
+        ),
+
+        .target(
+            name: "SentryCrash/Reporting/Tools",
+            path: "Sources/SentryCrash/Reporting/Tools",
+            publicHeadersPath: "."
+        ),
+
+        // TODO: make tests work.
```

Comment:
It’d be nice to run tests from the command-line using `swift test`, but I didn’t have time today to figure it out as (gracefully?) as the rest.